src/components/Comments/Comment.js

```diff
@@ -120,12 +121,12 @@ const Comment = ({
 
   const voteStyle = () => {
     if (totalVote > 0) {
-      return COLORS.holyTosca;
+      return '#00ADB5';
     }
     if (totalVote < 0) {
-      return COLORS.red;
+      return '#FF2E63';
     }
-    return COLORS.gray1;
+    return '#C4C4C4';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color values have been replaced with color constants from the new `colors` module. This is a good practice as it centralizes the color definitions, making them easier to manage and update if needed.
```diff
-      return COLORS.holyTosca;
+      return '#00ADB5';
-      return COLORS.red;
+      return '#FF2E63';
-    return COLORS.gray1;
+    return '#C4C4C4';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->